package.json

```diff
@@ -23,12 +23,16 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "bignumber.js": "^9.0.1",
     "bn.js": "^5.2.0",
     "buffer": "^6.0.3",
+    "cbor": "^7.0.5",
     "class-transformer": "^0.4.0",
     "class-validator": "^0.13.1",
     "events": "^3.3.0",
     "js-sha256": "^0.9.0",
+    "multibase": "^4.0.4",
+    "multihashes": "^4.0.2",
```

Comment:
Just reverted back to multibase@4.0.2, since 4.0.3+ does not work anymore in React Native according to [js-multibase#98](https://github.com/multiformats/js-multibase/issues/98).